frontend/src/pages/Classrooms/Select/index.tsx

```diff
@@ -9,15 +9,19 @@ import { Table, TableRow, TableCell } from "@/components/Table";
 import EmptyDataBanner from "@/components/EmptyDataBanner";
 import Button from "@/components/Button";
 import { MdAdd } from "react-icons/md";
+import { OrgRole } from "@/types/users";
 
 const ClassroomSelection: React.FC = () => {
   const [classrooms, setClassrooms] = useState<IClassroom[]>([]);
+  const [orgRole, setOrgRole] = useState<OrgRole>(OrgRole.MEMBER);
   const orgID = useUrlParameter("org_id");
   const [loading, setLoading] = useState(true);
   const { setSelectedClassroom } = useContext(SelectedClassroomContext);
 
   const navigate = useNavigate();
 
+  //TODO: check if the user is an org admin before showing create classroom
```

Comment:
Make sure this ticketed and attach ticket number.

Reply:
this is actually done. I just forgot to remove the TODO